packages/atomic/src/components/atomic-results-per-page/atomic-results-per-page.tsx

```diff
@@ -33,6 +34,7 @@ export class AtomicResultsPerPage implements InitializableComponent {
   private resultPerPage!: ResultsPerPage;
   public searchStatus!: SearchStatus;
   private choices!: number[];
+  private readonly radioGroupName = randomID('atomic-results-per-page-');
```

Comment:
With server-side rendering, the radio group name created on the server will differ from the one in browser. If so, could this break accessibility? Or maybe Stencil can recover and resolve the difference?

Reply:
I don't think so, the only reason I added a `name` is so that when we select a radio button, the others are deselected.